lib/carto/named_maps/template.rb

```diff
@@ -183,7 +183,11 @@ def attributes_and_interactivity(layer_infowindow, layer_tooltip)
           }
         end
 
-        [attributes, interactivity.join(',')]
+        if interactivity.empty?
+          interactivity << 'cartodb_id'
```

Comment:
One small caveat and in line with what you're doing in line 174:

```ruby
if interactivity.empty?
  interactivity = ['cartodb_id']
end
```

Or, alternatively, changing 174 to add with `<<`, but I like you're style better so I'd just change this last one.

Sorry to be nitpicky, but it's the kind of thing that confuses an new reader to this code!